src/pages/reactivistas/AddParticipantForm.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, FormEvent } from 'react';
+import { ReactGroup } from '../../lib/types';
+
+export const AddParticipantForm = ({ group }: { group: ReactGroup }) => {
+  const [discordUser, setDiscordUser] = useState('');
+  const [isSuccess, setIsSuccess] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+
+  const onAddParticipantSubmit = async (
+    event: FormEvent<HTMLFormElement>,
+    discordUser: string,
+    id: string,
+  ) => {
+    event.preventDefault();
+
+    const data = {
+      discordUser: discordUser,
+      id: id,
+    };
+
+    try {
+      setIsSuccess(true);
```

Comment:
`isSuccess` should be set to true only if `res` returns properly, which means that the POST request worked. I would set `isLoading` here.

Reply:
You can check for similar logic on the `CreateGroupForm` component.